### PR TITLE
Review tests

### PR DESCRIPTION
**Pull request by ChrisSimmons**
I know almost nothing about Node, but these tests look to me like there might be issues.  Trying some things to test my theory.

### CODE REVIEW

File: tests/searchimagescreativetests.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ test.beforeEach(t=>{
             .reply(200, {response : "response"})
             .get("/v3/search/images/creative")
             .query({ "page": 3, "phrase": "cat" })
-            .reply(200, {response : "response"})
```
Comment (ChrisSimmons) [inline tests/searchimagescreativetests.js:66]: Every combination here just returns `{response : "response"}` so it seems that the tests have a high chance of false passes.

These changes highlight the rough basics of what I think needs to happen.

Comment (mapitman) [reply tests/searchimagescreativetests.js:66]: @ChrisSimmons  Any plans to continue this branch or should we close this and delete the branch?

**Comment by mapitman** [comment]
@ChrisSimmons Any plans to continue this branch or should we close this and delete the branch?

**Comment by mapitman** [comment]
I'm working on another fix where I'll handle this change. Closing.

**Pull request closed without merging**